src/ScriptCs.Engine.Roslyn/ScriptCs.Engine.Roslyn.csproj

```diff
@@ -15,7 +15,7 @@
     <Compile Include="..\ScriptCs.Core\Guard.cs" Link="Guard.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Scripting" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Scripting" Version="3.6.0" />
```

Comment:
Why add this here and not in `Directory.Build.targets`?

Reply:
of course you are right - I updated from VS and it injected the versions here. corrected now